### ISSUE TITLE
Unable to select mods until restart

### ISSUE THREAD

**Issue by tbennett6421** [spelunky-fyi/modlunky2#513]
Happened on update:

In the install mods tab: after clicking browse under source and selecting any file makes the form unusable with the following error. I've confirmed this happens when either successfully picking a zip file or just picking any bogus file. This is also the case if you want to install more then one mod at a time.

Clicking the form opens the file wizard successfully once, but then this issue occurs until the program is restarted. Changing tabs has no impact on the issue.

```
2022-12-29 20:46:00,298: Running on http://127.0.0.1:9526 (CTRL + C to quit)
2022-12-29 20:46:21,405: Worker: Stopped seeing pings. Shutting down...
2022-12-29 20:46:25,541: Worker process went away... Restarting it.
2022-12-29 20:46:34,748: Extracting MyMod1.zip to C:\Games\Vendors\Steam\steamapps\common\Modlunky 2\Mods\Packs\MyMod1
2022-12-29 20:46:34,757: Finished installing MyMod1.zip to C:\Games\Vendors\Steam\steamapps\common\Modlunky 2\Mods\Packs\MyMod1
2022-12-29 20:46:43,756: Unhandled Exception: Traceback (most recent call last):
  File "tkinter\__init__.py", line 1892, in __call__
  File "modlunky2\ui\install.py", line 59, in browse
AttributeError: 'str' object has no attribute 'exists'
```

Unsure of what version I was running before... No version info in the program that I can see.
See the following

```sh
$ openssl md5 modlunky2.*
MD5(modlunky2.backup.exe)= 830e688549bf8f9adde6486c1c1a9ae3
MD5(modlunky2.exe)= 9e36a943f3359d40936c7c3ef38e6d47
$ openssl sha1 modlunky2.*
SHA1(modlunky2.backup.exe)= fce108fbbd8d7d1a9804a7e61f901c7f712c1c51
SHA1(modlunky2.exe)= 7409d79025f45218ebb1a107f6aa1147d2990753
```

**Comment by gmjosack** [comment]
Not sure why this could have broken recently as this code hasn't changed in a year but I've pushed a fix 4699efb370366de74cc8e08058c7f1fb975cfe46 and it should be resolved in the next release. Thanks for the detailed report!

**Comment by mauvealerts** [comment]
Version 0.28.1 has the fix
